src/utils/test/configuration.cpp

```diff
@@ -70,6 +70,7 @@ TEST(configuration, load)
     ASSERT_TRUE(c->load("config-sample.ini"));
 }
 
+/**
```

Comment:
Why disable this test case?

Reply:
Sorry I have no idea why these code is disabled. Maybe for test use. 
fixed.